lightning-tag/detailed-reference.md

```diff
@@ -133,6 +133,17 @@ The following tasks occur when `page()` is called:
 1. A new page view registers in your Lotame DMP.
 1. Targeting executes, and audiences return on either the callback method or localStorage (or both) depending on how you configured audience extraction on your site.
 
+### getAudiences()
+
+`getAudiences(limit)` is a helper method. It is a shortcut that retrieves the targeted audience array from `localStorage`.
+
+```javascript
+window.lotame_<lotameClientId>.getAudiences(limit);
+```
+
+Audiences will be returned as an array of targeting codes or audience IDs, based on your established tag settings. Example: `["aud1","aud2"]` or `["123","789"]`. If you would like to limit the number of audiences returned, send in a limit like `getAudiences(100)`.
```

Comment:
is the limit optional?  this paragraph sounds optional but every example shows the limit?